app/controllers/publications_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PublicationsController < ApplicationController
+
+  def index
+    @albums = Publication.find_votes("album")
+    @books = Publication.find_votes("book")
+    @movies = Publication.find_votes("movie")
+  end
+
+  def show
+    @publication = Publication.find(params[:id])
+  end
+
+  def create
+    @publication = Publication.new(publication_params)
```

Comment:
What about errors/flash notices here and in the `update`, `destroy` etc actions?